imsc1/spec/ttml-ww-profiles.html

```diff
@@ -4609,34 +4611,62 @@ <h2>WCAG and MAUR Considerations</h2>
-      as a <em>functional</em> replacement for the applied style.</p>
-
-      <p>For instance, in subtitling and captioning, italics can be used to indicate an off screen speaker context (for example a
-      voice from a radio). An author can choose to include this functional information in the text equivalent; for example, by
-      including the word "Radio: " before the image equivalent text. Note that images in an <a>Image Profile</a> <a>Document
-      Instance</a> that are intended for use as <em>captions</em>, i.e. intended for a hard of hearing audience, might already
-      include this functional information in the rendered text.</p>
-
-      <p><a data-cite="!WCAG21#text-alternatives">Guideline 1.1</a> of [[!WCAG21]] also recommends that accessible <a data-lt="Text Alternative">Text Alternatives</a> must be
-      "programmatically determinable." This means that the text must be able to be read and used by the assistive technologies (and
-      the accessibility features in browsers) that people with disabilities use. It also means that the user must be able to use
-      their assistive technology to find the alternative text (that they can use) when they land on the non-text content (that they
-      can't use).</p>
+      <section class='appendix' id='wcag-criterion-1-1-1'>
+        <h4>Success Criterion 1.1.1 Non-text Content</h4>
+
+        <p><a data-cite="!WCAG21#non-text-content">Success Criterion 1.1.1</a> at [[!WCAG21]] specifies that, with some exceptions,
+        all <a>non-text content</a> that is presented to the user has a <a>text alternative</a> that serves the equivalent
+        purpose.</p>
+
+        <p>In the context of this specification, <a>text alternative</a> is intended primarily to support users of the subtitles who
+        cannot see images (the <a>non-text content</a>). Since the images of an <a>Image Profile</a> <a>Document Instance</a>
+        usually represent subtitle or caption text, the guidelines for authoring text equivalent strings given at <span
+        class='sec-no'>Images of text</span> of [[HTML5]] are appropriate.</p>
+
+        <p>Thus, for each subtitle in an <a>Image Profile</a> <a>Document Instance</a>, a text equivalent content in a <a>Text
+        Profile</a> <a>Document Instance</a> SHOULD be written so that it conveys all essential content and fulfills the same
+        function as the corresponding subtitle image. In the context of subtitling and captioning, this content will be (as a
+        minimum) the verbatim equivalent of the image without précis or summarization. However, the author MAY include extra
+        information to the text equivalent string in cases where styling is applied to the text image with a deliberate connotation,
+        as a <em>functional</em> replacement for the applied style.</p>
+
+        <aside class="example">In subtitling and captioning, italics can be used to indicate an off screen speaker context (for

```

Comment:
I think this is too suggestive that use of italics in this way is a globally accepted practice.

```suggestion
       <aside class="example">Consider a subtitling and captioning practice that uses italics to indicate an off screen speaker context (for
```

Reply:
See revised text.

Reply:
Thanks, that works for me.